swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -1749,7 +1741,7 @@ module Exprs {
 
   module Conversions {
     class ConversionOrIdentity =
-      Synth::TIdentityExpr or Synth::TExplicitCastExpr or Synth::TImplicitConversionExpr;
+      Synth::TIdentityExpr or Synth::TExplicitCastExpr or Synth::TImplicitConversionExpr or Synth::TInOutExpr;
```

Comment:
Oh, I guess that's also a way to do it 😄. What I actually had in mind (and maybe I should've been more clear about that. Sorry!) is to implement `convertsFrom` on the `InOutExpr` class like we do for `IdentityExpr` [here](https://github.com/github/codeql/blob/main/swift/ql/lib/codeql/swift/elements/expr/IdentityExpr.qll#L4).

Reply:
I did that already - I think they're both necessary.

Reply:
Alright, let's keep it like this for now then. I can't really think of why they should be necessary, but that's not such a big deal.